src/main/resources/liquibase/crop_changelog/14_1_0.xml

```diff
@@ -293,7 +293,7 @@
     </changeSet>
 
 	<changeSet author="nahuel" id="v14.1.0-1">
-		<preConditions>
+		<preConditions onFail="MARK_RAN">
```

Comment:
@vmaletta not sure if it was @nahuel-soldevilla intention to use HALT as default for this changeset. 
Please confirm

Reply:
mmm :thinking: , I think I just forgot to add it. We'll need to delete the changelogs

@clarysabel @vmaletta

Reply:
Ok, It will be better to include this change in a different branch, so I will not have problems running the jenkins jobs (tests and everything else for this functional ticket that needs to pass branch testing) @vmaletta Please revert this change in this branch

Reply:
Ok